test/docker-volume-test.sh

```diff
@@ -0,0 +1,137 @@
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "import os; print(os.listdir('/mnt'))" | egrep "foo.*bar|bar.*foo"
+docker volume rm named_vol
+fold_end test.docker.3
+
+fold_start test.docker.4 "docker-volume: read only -v option for directory"
+
+
+MNT_SRC=$PWD/host_dir
+MNT_DST=/mnt
+mkdir -p $MNT_SRC
+touch    $MNT_SRC/foo
+touch    $MNT_SRC/bar
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST:ro \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "import os; print(os.listdir('/mnt'))" | egrep "foo.*bar|bar.*foo"
+
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST:ro \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "f=open('${MNT_DST}/hello.txt', 'w'), print('hello',file=f);close(f)" 2> fail_log || true
+cat fail_log
+cat fail_log | grep "OSError" >/dev/null
+rm -rf $MNT_SRC
+
+2> fail_log || true
+
+fold_end test.docker.4
+
+fold_start test.docker.5 "docker-volume: read only -v option for file"
+MNT_SRC=/tmp/hello.txt
+MNT_DST=/mnt/hello.txt
+echo "hello_world" > $MNT_SRC
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST:ro \
+       -e RUNU_AUX_DIR=$RUNU_AUX_DIR alpine /bin/busybox cat $MNT_DST | grep 'hello_world'
+
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST:ro \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "f=open('${MNT_DST}', 'w'), print('hello',file=f);close(f)" 2> fail_log || true
+cat fail_log
+cat fail_log | grep "OSError" >/dev/null
+rm -f $MNT_SRC
+fold_end test.docker.5
+
+fold_start test.docker.6 "docker-volume: read only -v option for named volume"
+
+docker run --rm -v named_vol:/mnt alpine touch /mnt/foo /mnt/bar
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v named_vol:/mnt:ro \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "import os; print(os.listdir('/mnt'))" | egrep "foo.*bar|bar.*foo"
+
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v named_vol:/mnt:ro \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "f=open('/mnt/hello.txt', 'w'), print('hello',file=f);close(f)" 2> fail_log || true
+cat fail_log
+cat fail_log | grep "OSError" >/dev/null
+
+docker volume rm named_vol
+
+fold_end test.docker.6
+
+else
+
+fold_start test.docker.0 "docker-volume: -v option should fail on ${TRAVIS_OS_NAME}"
+
+    MNT_SRC=$PWD/host_dir
+    MNT_DST=/mnt
+    mkdir -p $MNT_SRC
+    touch    $MNT_SRC/foo
+    touch    $MNT_SRC/bar
+    docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+           -e LKL_CONFIG=/lkl.json \
+           -e LKL_ROOTFS=imgs/python.img \
+           -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST \
```

Comment:
Because LKL_ROOTFS and LKL_USE_9PFS are both specified, isn't it always failed even on Linux ?